File: src/nl/rubensten/texifyidea/templates/LatexLiveTemplateProvider.java
package nl.rubensten.texifyidea.templates;

import com.intellij.codeInsight.template.impl.DefaultLiveTemplatesProvider;
import nl.rubensten.texifyidea.util.Constants;
import org.jetbrains.annotations.Nullable;

/**
 * @author Sten Wessel
 */
public class LatexLiveTemplateProvider implements DefaultLiveTemplatesProvider {

    @Override
    public String[] getDefaultLiveTemplateFiles() {
        return Constants.EMPTY_STRING_ARRAY;
    }

    @Nullable
    @Override
    public String[] getHiddenLiveTemplateFiles() {
        return new String[] {
                "liveTemplates/hidden/LaTeX"
        };
    }
}

File: src/nl/rubensten/texifyidea/templates/LatexLiveTemplateProvider.kt
package nl.rubensten.texifyidea.templates

import com.intellij.codeInsight.template.impl.DefaultLiveTemplatesProvider

/**
 * @author Sten Wessel
 */
class LatexLiveTemplateProvider : DefaultLiveTemplatesProvider {

    override fun getDefaultLiveTemplateFiles() = emptyArray<String>()

    override fun getHiddenLiveTemplateFiles() = arrayOf("liveTemplates/hidden/LaTeX", "liveTemplates/hidden/Math")
}